CONTRIBUTING.md

```diff
@@ -2,50 +2,28 @@
 
 ## Commit Message Format
 
-Our commit message format is based on the one used by AngularJS.
-
-Each commit message consists of a **header** and a **body**.  The header has a special
-format that includes a **type**, a **scope** and a **subject**:
-
 ```
-<type>(<scope>): <subject>
-<BLANK LINE>
-<body>
+NEPT-[ticket_number]: [commit_description]
 ```
 
-The **header** is mandatory and the **scope** of the header is optional.
-
-Any line of the commit message cannot be longer 100 characters! This allows the message to be easier
+The commit description cannot be longer 100 characters! This allows the message to be easier
 to read on GitHub as well as in various git tools.
 
 ### Revert
-If the commit reverts a previous commit, it should begin with `revert: `, followed by the header of the reverted commit. In the body it should say: `This reverts commit <hash>.`, where the hash is the SHA of the commit being reverted.
-
-### Type
-Must be one of the following:
-
-* **feat**: A new feature
-* **fix**: A bug fix
-* **docs**: Documentation only changes
-* **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing
-semi-colons, etc)
-* **refactor**: A code change that neither fixes a bug nor adds a feature
-* **perf**: A code change that improves performance
-* **test**: Adding missing tests
-* **chore**: Changes to the build process or auxiliary tools and libraries such as documentation
-generation
+If the commit reverts a previous commit, it should begin with `revert: `, followed by the reverted commit.
 
-### Scope
-The scope could be any component the commit is affecting. Use the business components
-defined at https://webgate.ec.europa.eu/CITnet/confluence/display/NEXTEUROPA/WCM+Components.
-
-### Subject
-The subject contains succinct description of the change:
+### Commit Description
+The description contains succinct description of the change:
```

Comment:
"The description contains a succinct description of the change and should follow below rules:"